data-science-pipelines-operator/base/params.env

```diff
@@ -0,0 +1,8 @@
+IMAGES_APISERVER=registry.access.redhat.com/rhods/odh-ml-pipelines-api-server-rhel8:latest
```

Comment:
Other components have the images published in registry.redhat.io/rhods/...  , not registry.access.redhat.com  (see RHODS-6299).  Also, it's possible that the images should be referenced by sha256, not tag

I believe you have to add them to the related_images info for the bundle bundle

Can you provide more info @dchourasia @zdtsw   @pablofelix ?

Reply:
yeah, think we should use `registry.redhat.io` which points to the production registery.

Reply:
thanks folks, I have updated the values to point to `registry.redhat.io` instead. 

I've kept the tags to `latest`, but do keep in mind that these are _default_ images, during builds the images used should be passed in via env vars and then into the kfdefs much like for [model-mesh](https://github.com/red-hat-data-services/odh-deployer/blob/main/kfdefs/rhods-model-mesh.yaml#L9-L22). 

The values in `params.env` are the values used when they are not specified in the kfdef. 

@rimolive has a PR going [here](https://github.com/red-hat-data-services/odh-deployer/pull/308/files#), and he will be populating the parameters soon, which should then be substituted much [like in modelmesh](https://gitlab.cee.redhat.com/service/managed-tenants-bundles/-/blob/main/addons/rhods/main/1.24.0/manifests/rhods-operator.1.24.0.clusterserviceversion.yml#L635-645).

Reply:
Thanks @HumairAK !